### ISSUE TITLE
Invalid BigQuery temp location

### ISSUE THREAD

**Issue by nevillelyh** [spotify/scio#549]
Trying this in the 0.3.0-beta3 repo.

```scala
:scioOpts --runner=DataflowRunner --project=scio-playground --zone=us-east1-d --gcsTempLocation=gs://scio-playground-us/scio --tempLocation=gs://scio-playground-us/scio
:newScio

// some one-liner job that reads from BQ
[main] INFO com.spotify.scio.bigquery.BigQueryClient - Existing destination table: scio-playground:scio_bigquery_staging_us.scio_query_20170417155429_1022205266
java.lang.IllegalArgumentException: BigQuery temp location expected a valid 'gs://' path, but was given '/var/folders/hd/f_wgpmsj3h1cv071xszfmyzh0000gn/T/scio-temp-1412900320157110831'
  at org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO$Read$Bound.validate(BigQueryIO.java:662)
  at org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO$Read$Bound.validate(BigQueryIO.java:507)
  at org.apache.beam.sdk.Pipeline.applyInternal(Pipeline.java:419)
  at org.apache.beam.sdk.Pipeline.applyTransform(Pipeline.java:350)
  at org.apache.beam.sdk.values.PBegin.apply(PBegin.java:58)
  at org.apache.beam.sdk.Pipeline.apply(Pipeline.java:172)
  at com.spotify.scio.ScioContext.applyInternal(ScioContext.scala:401)
  at com.spotify.scio.ScioContext$$anonfun$bigQuerySelect$1.apply(ScioContext.scala:479)
  at com.spotify.scio.ScioContext$$anonfun$bigQuerySelect$1.apply(ScioContext.scala:474)
  at com.spotify.scio.ScioContext.requireNotClosed(ScioContext.scala:358)
  at com.spotify.scio.repl.ReplScioContext.requireNotClosed(ReplScioContext.scala:37)
  at com.spotify.scio.ScioContext.bigQuerySelect(ScioContext.scala:473)
  at com.spotify.scio.ScioContext.typedBigQuery(ScioContext.scala:555)
  ... 32 elided
```

So somehow the `tempLocation` was not picked up.

**Comment by nevillelyh** [comment]
Looks like `:scioOpts` might be broken.

**Comment by nevillelyh** [comment]
Fixed in cd8b41fd55e44327048b6a6c0418cfe8b60084b2
